docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,124 @@
+# Release 381 (11 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
+* Fix regression in file-based system and connector access controls. ({issue}`12204`)
+
+## Web UI
+
+* Ensure consistent sort order in the list of workers. ({issue}`12290`)
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Delta Lake connector
+
+## Druid connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Elasticsearch connector
+
+* Fix failure when reading timestamp values with `row` type that are not ISO
+  8601 formatted. ({issue}`12250`)
+
+## Google Sheets connector
+
+## Hive connector
+
+## Iceberg connector
+
+* Add support for {doc}`/sql/update`. ({issue}`12026`)
+* Fix potential query failure or incorrect results when reading Iceberg table
+  containing equality deletion files. ({issue}`12026`)
+
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+## Kudu connector
+
+## Local file connector
+
+## MariaDB connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Oracle connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## Phoenix connector
+
+## Pinot connector
+
+## PostgreSQL connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12225`)
+
+## Prometheus connector
+
+## Raptor connector
+
+## Redis connector
+
+* Add `redis.table-description-cache-ttl` configuration property to set the
+  refresh interval for fetching table descriptions. ({issue}`12240`)
+
+## Redshift connector
+
+* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
+
+## SingleStore (MemSQL) connector
```

Comment:
add
```
* Prevent data loss when non-transactional insert fails. ({issue}`12229`)
```

Reply:
Ok .. added .. but question .. are you sure? How can we tell ourselves in the future. It seem that the SingleStore connector does not extend the JdbcPlugin .. which is what we used as criteria to create the list..